platform/javascript/os_javascript.cpp

```diff
@@ -440,12 +440,7 @@ void OS_JavaScript::initialize(const VideoMode &p_desired, int p_video_driver, i
 	visual_server = memnew(VisualServerRaster(rasterizer));
 	visual_server->init();
 
-	/*AudioDriverManagerSW::get_driver(p_audio_driver)->set_singleton();
-
-	if (AudioDriverManagerSW::get_driver(p_audio_driver)->init()!=OK) {
-
-		ERR_PRINT("Initializing audio failed.");
-	}*/
+	AudioDriverManagerSW::initialize(p_audio_driver);
```

Comment:
Does it work? If those were commented out I assume it means that audio was not working in 2.1 JavaScript?

Reply:
Forgot to comment it out after updating it :-1: